trollflow2/tests/test_trollflow2.py

```diff
@@ -844,6 +844,11 @@ def test_create_scene(self):
 class TestLoadComposites(TestCase):
     """Test case for loading composites."""
 
+    @pytest.fixture(autouse=True)
+    def inject_fixtures(self, caplog):
+        """Inject fixtures."""
+        self._caplog = caplog
```

Comment:
What is the benefit of storing a `_caplog` attribute and using that, rather than using the `caplog` attribute in the tests directly?

Reply:
This is the only way to use fixtures in `unittest.TestCase` classes.